recipes/adlfs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.3.0" %}
+
+package:
+  name: adlfs
+  version: {{ version }}
+
+source:
+  fn: adlfs-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/a/adlfs/adlfs-{{ version }}.tar.gz
+  sha256: 71862ccad2d9e37f68b24efd9fb72d8c930667c8abf99fa73df0d3373c00d129
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
```

Comment:
I think this section should basically match the "install_requires" in setup.py, since it has the same role (https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run)